manual/development.md

```diff
@@ -3,16 +3,14 @@
 Use a rake task to generate a cop template.
 
 ```sh
-$ bundle exec rake new_cop[Category/Name]
-created
-- lib/rubocop/cop/category/name.rb
-- spec/rubocop/cop/category/name_spec.rb
-modified
-- `require 'lib/rubocop/cop/category/name'` added into lib/rubocop.rb
```

Comment:
It reflects the result of the current cop generator. However, changing cop generator to display this `modified ` line, may make it easier for developers to understand.

I'd like to change cop generator with another PR so that it also displays `modified` line if necessary.
What do you think?

Reply:
Yeah, that would be nice.

Reply:
Thank you! I changed it with #4743.